packs/data/menace-under-otari-bestiary.db/animated-armor-bb.json

```diff
@@ -259,7 +259,7 @@
                 "value": ""
             },
             "hardness": {
-                "value": null
+                "value": 9
```

Comment:
We should make this line up with hazards and vehicles (`attributes.hardness`, no `value` object)